DEVELOP.md

```diff
@@ -84,63 +82,30 @@ Their upstream repos are assumed to have the remote name `origin`.
       version. In the "Starting a new version" section, the link will be added
       again for the new version.
 
-5.  Edit the README file:
```

Comment:
Actually, those variables are in the README as I have it in another PR.  Shouldn't we be keeping the original item 5?

Reply:
We agreed that we would no longer have the versons etc in the README file, so this one is resolved.